tim_miller/test/server_test.js

```diff
@@ -0,0 +1,63 @@
+var chai = require('chai');
+var chaihttp = require('chai-http');
+chai.use(chaihttp);
+var expect = chai.expect;
+var server = require(__dirname + '/../server.js');
+var fs = require('fs');
+
+
+describe('our express app', function() {
+
+  describe('receiveing a post request', function() {
+
+
+    before(function(done) {
+      this.fileContents = '';
+      fs.readFile(__dirname + '/../data/database.json', function(err, data) {
+        this.fileContents = data;
+        done();
+      });
```

Comment:
You should be creating a file before tests rather than reading one (then delete it after). The way you have it now makes your tests dependent on you having a file you created, but they should be able to run as independently as possible.